subprojects/tooling-api/tooling-api.gradle.kts

```diff
@@ -70,6 +70,7 @@ testFixtures {
     from(":logging")
     from(":dependencyManagement")
     from(":ide")
+    from(":languageJava")
```

Comment:
We should invert this - The test belongs in the java project.

Reply:
Will do.

Reply:
Done in 54eb122399a.